### ISSUE TITLE
Support tid parameter

### ISSUE THREAD

**Issue by tacoo** [carterjones/signalr#7]
This one is easy. Add `tid` at `makeURL`.

I am not good at golang but code will be something like this.
```
if u.Scheme != "http" && u.Scheme != "https" {
	params.Set("tid", fmt.Sprintf("%.0f", math.Floor(rand.Float64()*11)))
}
```

**Comment by carterjones** [comment]
Thank you for opening this issue and for the sample code.

I implemented this in ed40a9ba0a1dcb136b978662331710a389db8430.

Is this what you were looking for?

**Comment by tacoo** [comment]
Yes! This is great! Thank you
